game.js

```diff
@@ -467,32 +479,28 @@ class Game {
             if (config.DEBUG) console.log("Starting disconnect Timeout in Game " + this.game_id + " gameOver()");
             this.disconnect_timeout = setTimeout(() => {  this.conn.disconnectFromGame(this.game_id);  }, 1000);
         }
-    } /* }}} */
-    header() { /* {{{ */
+    }
+    header() {
         if (!this.state)  return;
-        let color = 'W  ';  // Playing white against ...
-        let player = this.state.players.black;
-        if (player.username === config.username) {
-            player = this.state.players.white;
-            color = '  B';
-        }
-        let name = player.username;
-        let handi = (this.state && this.state.handicap ? "H" + this.state.handicap : "  ");
-        return sprintf("%s %s  [%ix%i]  %s", color, name, this.state.width, this.state.width, handi);
+        const botIsBlack = this.state.players.black.username === config.username;
+        const color = botIsBlack ? '  B' : 'W  ';  // Playing black / white against ...
+        const player = botIsBlack ? this.state.players.white : this.state.players.black;
+        const handi = (this.state && this.state.handicap ? "H" + this.state.handicap : "  ");
+        return sprintf("%s %s  [%ix%i]  %s", color, player.username, this.state.width, this.state.width, handi);
```

Comment:
Consider replacing printf with string interpolation
```
return `${color} ${player.username}  [${this.state.width}x${this.state.width}]  ${handi}`
```

Reply:
i didnt know we could do that :)
yes i also think its a great idea :)
this is standard